frigate/http.py

```diff
@@ -386,6 +386,18 @@ def set_sub_label(id):
             400,
         )
 
+    if new_score > 1.0 or new_score < 0:
+        return make_response(
+            jsonify(
+                {
+                    "success": False,
+                    "message": new_score
+                    + " does not fit within the expected bounds 0 < score < 1.0",
```

Comment:
`<= 1.0`

Reply:
This is the case where we throw an error, a score of 100% or 0% is acceptable so it should not include those

Reply:
> expected bounds 0 < score < 1.0
I understand this as the expected values are greater (not equal) than 0 and less than 1. I'm wrong?

Reply:
by the way if you want to suggest code directly you can select code and use the suggest button

<img width="797" alt="Screenshot 2023-07-10 at 6 57 05 PM" src="https://github.com/blakeblackshear/frigate/assets/14866235/2bdaf9c5-8d36-4838-b56f-98d9d72d9f6a">